Sources/RingPublishingTracking/Public/RingPublishingTrackingConfiguration.swift

```diff
@@ -31,6 +31,10 @@ public struct RingPublishingTrackingConfiguration {
     /// Default ad space name of the application, for example "ads/list/sport"
     /// If no new value will be set during app session, this value will be always used.
     public let applicationDefaultAdvertisementArea: String
+    
+    /// Application site name, for example site name like "onet" or "newsweek".
```

Comment:
lepiej może coś w stylu coś w stylu:

Application advertisement site, which is used as a prefix for all advertisement areas